### PR TITLE
[PEPPER-468] Updates parent or parent(s) to parent(s)/guardian(s) for somatic assent.

### PR DESCRIPTION
**Pull request by otchet-broad**
## Context

(PEPPER-468)[https://broadworkbench.atlassian.net/browse/PEPPER-468] creates a new revision of variables used on in the Osteo Somatic Assent activity and replaces parent or parent(s) with parent(s)/guardian(s).

**Before**:
Screenshot 2023-05-05 at 1 06 28 PM
Screenshot 2023-05-05 at 1 06 54 PM

**After**:
Screenshot 2023-05-08 at 10 00 41 AM
Screenshot 2023-05-08 at 10 00 56 AM
Screenshot 2023-05-08 at 10 01 30 AM

Deployment:
To deploy in each environment:
In pepper-apis/studybuilder-cli:

Use render.sh to build output-config/application.conf for the target environment.
Run: java -Dconfig.file=output-config/application.conf -jar /path/to/StudyBuilder.jar --vars output-config/vars.conf --substitutions ./studies/osteo/subs.conf ./studies/osteo/study.conf --run-task OsteoSomaticAssentV3

## Checklist

- [X] I have labeled the type of changes involved using the `C-*` labels.
- [X] I have assessed potential risks and labeled using the `R-*` labels.
- [X] I have considered error handling and alerts, and added `L-*` labels as needed.
- [X] I have considered security and privacy, and added `I-*` labels as needed
- [X] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [X] N/A If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [X] N/A If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [X] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [] They are user-visible in dev as a regular user journey and require no additional instructions.
- [X] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [X] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [X] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

**Comment by ssettipalli** [comment]
looks ok to me

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java
Comment (aweng98) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java:172]: I like this. Are variable names not unique in a study? Yet, this still provides a good protection in case of accidental duplicated variable names.

**Review by aweng98 — APPROVED** [review]
LGTM. I can reuse this class for pepper-449. Thank you.

**Review by rfricke-asymmetrik — APPROVED** [review]
LGTM

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java
Comment (aweng98) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java:102]: `handle` is not used.

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java
Comment (otchet-broad) [reply pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoSomaticAssentV3.java:172]: I found this code in an earlier PR for a similar task and thought it did a good job of helping to narrow the scope of the change I needed to make.

**Review by ssettipalli — CHANGES_REQUESTED** [review]
@otchet-broad 
I believe you need to combine https://broadworkbench.atlassian.net/browse/PEPPER-460 into this one because its one activity (CONSENT_ADDENDUM_PEDIATRIC)  that covers both somatic consent addendum and assent addendum

**Comment by otchet-broad** [comment]
@ssettipalli - Is there a technical reason 460 can't be done independently?  Wouldn't doing these this way be making incremental changes to the same activity?

**Comment by otchet-broad** [comment]
@ssettipalli: Is there a technical reason 460 can't be done independently?  Wouldn't doing these this way be making incremental changes to the same activity?

**Comment by ssettipalli** [comment]
@otchet-broad . It can be done separately too and need to be done little differently then.
That will be dependent on this ticket being applied. Thats fine. I will handle it in 460. I need to wait until you merge and apply which is ok.
Go ahead and merge this if you'r ready

**Review by ssettipalli — APPROVED** [review]

**Pull request merged**
